src/fetch/index.ts

```diff
@@ -71,9 +71,18 @@ export const edenFetch =
             } catch (error) {}
 
         const fetch = config?.fetcher || globalThis.fetch
-        const queryStr = query
-            ? `?${new URLSearchParams(query).toString()}`
+
+        const nonNullishQuery = query
+            ? Object.fromEntries(
+                  Object.entries(query).filter(
+                      ([_, val]) => val !== undefined && val !== null
+                  )
+              )
+            : null
```

Comment:
```suggestion
        const nonNullishQuery = query
            ? Object.fromEntries(
                  Object.entries(query).filter(
                      ([_, val]) => val != null)
                  )
              )
            : null
```

js quirk: `!=` null catches both undefined and null

Reply:
Unless for performance reason, I dont think relying on implicit behaviours is good